browserstack.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Is there any way to get this file into the `example`/`test` folder?  Having it in the root directory implies that it matters to normal usage of the library.  If that turns out to be difficult, it's not a big deal.